diff -up kdepim-runtime-20.03.80/resources/ews/ewsclient/auth/ewsoauth.cpp.omv~ kdepim-runtime-20.03.80/resources/ews/ewsclient/auth/ewsoauth.cpp
--- kdepim-runtime-20.03.80/resources/ews/ewsclient/auth/ewsoauth.cpp.omv~	2020-03-23 03:58:34.214555796 +0100
+++ kdepim-runtime-20.03.80/resources/ews/ewsclient/auth/ewsoauth.cpp	2020-03-23 03:58:39.161565072 +0100
@@ -163,7 +163,7 @@ void EwsOAuthUrlSchemeHandler::requestSt
 
 void EwsOAuthReplyHandler::networkReplyFinished(QNetworkReply *reply)
 {
-#if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
     const auto networkError = reply->error();
 #else
     const auto networkError = reply->networkError();
diff -up kdepim-runtime-20.03.80/resources/ews/ewsclient/auth/ewspkeyauthjob.cpp.omv~ kdepim-runtime-20.03.80/resources/ews/ewsclient/auth/ewspkeyauthjob.cpp
--- kdepim-runtime-20.03.80/resources/ews/ewsclient/auth/ewspkeyauthjob.cpp.omv~	2020-03-23 03:59:54.803707222 +0100
+++ kdepim-runtime-20.03.80/resources/ews/ewsclient/auth/ewspkeyauthjob.cpp	2020-03-23 03:59:59.822716674 +0100
@@ -114,7 +114,7 @@ void EwsPKeyAuthJob::sendAuthRequest(con
 
 void EwsPKeyAuthJob::authRequestFinished()
 {
-#if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
     const auto networkError = mAuthReply->error();
 #else
     const auto networkError = mAuthReply->networkError();
diff -up kdepim-runtime-20.03.80/resources/tomboynotes/o2/o1.cpp.omv~ kdepim-runtime-20.03.80/resources/tomboynotes/o2/o1.cpp
--- kdepim-runtime-20.03.80/resources/tomboynotes/o2/o1.cpp.omv~	2020-03-23 03:55:48.246246391 +0100
+++ kdepim-runtime-20.03.80/resources/tomboynotes/o2/o1.cpp	2020-03-23 03:55:58.558265504 +0100
@@ -276,7 +276,7 @@ void O1::onTokenRequestFinished()
     qCDebug(TOMBOYNOTESRESOURCE_LOG) << "O1::onTokenRequestFinished";
     QNetworkReply *reply = qobject_cast<QNetworkReply *>(sender());
     reply->deleteLater();    
-#if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
     const auto networkError = reply->error();
 #else
     const auto networkError = reply->networkError();
@@ -367,7 +367,7 @@ void O1::onTokenExchangeFinished()
 
     QNetworkReply *reply = qobject_cast<QNetworkReply *>(sender());
     reply->deleteLater();
-#if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
     const auto networkError = reply->error();
 #else
     const auto networkError = reply->networkError();
diff -up kdepim-runtime-20.03.80/resources/tomboynotes/o2/o1..omv~ kdepim-runtime-20.03.80/resources/tomboynotes/o2/o1.
diff -up kdepim-runtime-20.03.80/resources/tomboynotes/o2/o2.cpp.omv~ kdepim-runtime-20.03.80/resources/tomboynotes/o2/o2.cpp
--- kdepim-runtime-20.03.80/resources/tomboynotes/o2/o2.cpp.omv~	2020-03-23 03:54:42.464124876 +0100
+++ kdepim-runtime-20.03.80/resources/tomboynotes/o2/o2.cpp	2020-03-23 03:54:52.662143664 +0100
@@ -279,7 +279,7 @@ void O2::onTokenReplyFinished()
 {
     qCDebug(TOMBOYNOTESRESOURCE_LOG) << "O2::onTokenReplyFinished";
     QNetworkReply *tokenReply = qobject_cast<QNetworkReply *>(sender());
-#if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
     const auto networkError = tokenReply->error();
 #else
     const auto networkError = tokenReply->networkError();
@@ -395,7 +395,7 @@ void O2::refresh()
 void O2::onRefreshFinished()
 {
     QNetworkReply *refreshReply = qobject_cast<QNetworkReply *>(sender());
-#if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
     const auto networkError = refreshReply->error();
 #else
     const auto networkError = refreshReply->networkError();
diff -up kdepim-runtime-20.03.80/resources/tomboynotes/o2/o2requestor.cpp.omv~ kdepim-runtime-20.03.80/resources/tomboynotes/o2/o2requestor.cpp
--- kdepim-runtime-20.03.80/resources/tomboynotes/o2/o2requestor.cpp.omv~	2020-03-23 03:53:50.958030260 +0100
+++ kdepim-runtime-20.03.80/resources/tomboynotes/o2/o2requestor.cpp	2020-03-23 03:53:56.287040026 +0100
@@ -82,7 +82,7 @@ void O2Requestor::onRefreshFinished(QNet
 void O2Requestor::onRequestFinished()
 {
     QNetworkReply *senderReply = qobject_cast<QNetworkReply *>(sender());
-#if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
     const auto networkError = senderReply->error();
 #else
     const auto networkError = senderReply->networkError();
diff -up kdepim-runtime-20.03.80/resources/tomboynotes/tomboyjobbase.cpp.omv~ kdepim-runtime-20.03.80/resources/tomboynotes/tomboyjobbase.cpp
--- kdepim-runtime-20.03.80/resources/tomboynotes/tomboyjobbase.cpp.omv~	2020-03-23 03:57:26.498429121 +0100
+++ kdepim-runtime-20.03.80/resources/tomboynotes/tomboyjobbase.cpp	2020-03-23 03:57:30.730437021 +0100
@@ -42,7 +42,7 @@ void TomboyJobBase::setAuthentication(co
 
 void TomboyJobBase::checkReplyError()
 {
-#if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
     const auto networkError = mReply->error();
 #else
     const auto networkError = mReply->networkError();
