--- kdepim-runtime-17.04.0/plugins/autotests/CMakeLists.txt.omv~	2017-04-22 15:42:11.600584502 +0200
+++ kdepim-runtime-17.04.0/plugins/autotests/CMakeLists.txt	2017-04-22 15:42:21.425636032 +0200
@@ -8,6 +8,7 @@ set( EXECUTABLE_OUTPUT_PATH ${CMAKE_CURR
 include_directories(
     ${kdepim-runtime_SOURCE_DIR}/plugins
     ${kdepim-runtime_BINARY_DIR}/plugins
+    ${kdepim-runtime_BINARY_DIR}/plugins/akonadi_serializer_mail_autogen/include
 )
 
 # convenience macro to add akonadi qtestlib unit-tests
